nautobot/extras/templatetags/plugins.py

```diff
@@ -80,6 +89,17 @@ def plugin_full_width_page(context, obj):
     return _get_registered_content(obj, "full_width_page", context)
 
 
+@register.inclusion_tag("extras/templatetags/plugin_nav_tabs.html", takes_context=True)
+def plugin_nav_tabs(context, obj):
```

Comment:
Wondering if `nav` here and above is misleading since normally `nav` is referring to the top menu bar. Maybe something like `plugin_object_detail_tabs` would be better? Not sure.